src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderResourceIntTest_transferIn.kt

```diff
@@ -287,134 +288,145 @@ class OffendersResourceTransferImpTest : ResourceTest() {
     @Nested
     @DisplayName("Failed transfer in")
     inner class Failed {
-      private lateinit var offenderNo: String
-
-      @BeforeEach
-      internal fun setUp() {
-        offenderNo =
-          OffenderBuilder().withBooking(
-            OffenderBookingBuilder(
-              prisonId = "LEI",
-              bookingInTime = LocalDateTime.now().minusDays(1),
-            ),
-          ).save(testDataContext).offenderNo
-      }
+      @Nested
+      inner class WhenInTransit {
```

Comment:
separated out to 2 nested test

- One for when has been transferred out
- The other when they haven't